dist-persist/wbstack/src/Internal/ApiWbStackInit.php

```diff
@@ -138,4 +140,20 @@ public function getAllowedParams() {
             ],
         ];
     }
+
+    public function initMainPage() {
+        $user = WbStackPlatformReservedUser::getUser();
+        $comment = \CommentStoreComment::newUnsavedComment('(automated) add example content');
+
+		$title = \Title::newMainPage();
+        $page = new \WikiPage($title);
```

Comment:
```suggestion
        $page = new \WikiPage( $title );
```

Reply:
is this formatting something we do now? i don't mind, just curious

Reply:
I usually try to stay consistent within a method which is why I raised the comment. I don't think we have any hard rules on this.

Reply:
that makes a lot of sense, I didn't notice :+1: